examples/typescript.ts

```diff
@@ -0,0 +1,14 @@
+// Import types
+import {MessagePublishedData} from '../src/google/events/cloud/pubsub/v1/data_pb';
+
+// Annotate objects with types
+const pubsubData: MessagePublishedData.AsObject = {
+    message: {
+        data: atob('Pub/Sub data'),
+        attributesMap: undefined,
+        messageId: 'my-message-id'
+    },
+    subscription: 'foo'
```

Comment:
Is subscription really any arbitrary string? Maybe the examples should be more representative of the data developers would see in practice.

Reply:
The subscription is

Here's one from Eventarc for example:

```js
'projects/serverless-com-demo/subscriptions/cre-us-central1-pubsub-trigger-5-sub-000'
```

I'll change it to something in this format.